src/classBased/components/InputTodo.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+
+export default class InputTodo extends Component {
+  state = {
+    title: ""
+  };
+  onChange = e => {
+    this.setState({
+      title: e.target.value
+    });
+  };
+  handleSubmit = e => {
+    e.preventDefault();
+    if (this.state.title.trim()) {
+      this.props.addTodoProps(this.state.title);
+      this.setState({
+        title: ""
+      });
+    } else {
+      alert("Please write your todo")
```

Comment:
You may consider using a custom message rather than a Javascript alert message. Window Alerts are used mainly for testing purposes.